### DISCUSSION TITLE
Will auto_defer parameter get added to View in Stable version?

### DISCUSSION THREAD

**Discussion by Jay-ArBrouillard** [nextcord/nextcord#612] [General]
I have a very specific use case heavily reliant on Selects. Select options are repeatibly used and sometimes the callback will fail with 404 Unknown Interaction. It doesn't break the Select, you simply have to wait a couple minutes and the Select is usable again.

I think if auto_defer is added, then I can catch these errors immediately instead of forcing user to wait until timeout is reached or nextcord throws error itself. I tried using latest development versions of nextcord but this was causing many other errors.

**Comment by ooliver1** [comment]
Of course it will, there are just other features that need to be finished before another version is made

**Reply by Jay-ArBrouillard** [reply]
Awesome. I guess I should have rephrased the question to when that release estimate would be?

**Reply by ooliver1** [reply]
No ETA, ETA's cause stress and are hard to meet, sorry

**Reply by ooliver1** [reply]
oh yeah by the way this was added in b1 which was a few days ago
